modules/special-rules/partials/template-placement-bullet.adoc

```diff
@@ -0,0 +1,8 @@
+. Place a Barrage template so that it:
+* is completely within range and line of sight of the firer
++
+NOTE: If the unit or weapon also has {artillery}, the template does not need to be in line of sight of the firer.
+
+////
+ Editing note, I've added 'or weapon' to the Artillery note for future-proofing purposes. For example if Carapace Landing Pads get added, and they allow one weapon that uses the Barrage template to fire with the Artillery rule.
```

Comment:
Good thinking, but I'd argue that it'd be cleaner and more consistent to treat Carapace Landing Pad just like most other special rules -- as overrides to the norm. We'd just need to be clear that in effect the CLP temporarily adds the Artillery ability to the weapon, which we should do anyway. Hence, no need to mention unit here.

Reply:
There are potentially other war engines that would have Artillery on a single weapon (backporting the Iron Warriors Ordinatus from EA, for example), but we can alway go back and tweak it if and when that happens.